packages/tpetra/core/test/CrsGraph/CrsGraph_UnitTests_Swap.cpp

```diff
@@ -125,55 +123,55 @@ TEUCHOS_STATIC_SETUP()
 // gbl_num_columns: Max # of columns in the matrix-representation of the graph.
 //                  This should be >= the highest value of v from all edges (u,v) in edges.
 //                  Note: u and v are 0-indexed, so if the highest v is 11, then this should be 12.
-template<class LO, class GO, class Node, class comm_t>
-Teuchos::RCP<Tpetra::CrsGraph<LO, GO, Node>>
-generate_crsgraph(Teuchos::RCP<comm_t>                      comm,
-                  const std::vector<std::pair<GO, GO>>&     gbl_edges,
-                  const std::vector<std::pair<GO, size_t>>& gbl_row_owners,
-                  const size_t                              gbl_num_columns,
-                  const bool                                do_fillComplete=true)
+template<class LO, class GO, class Node>
+Teuchos::RCP<Tpetra::CrsGraph<LO, GO, Node> >
+generate_crsgraph(Teuchos::RCP<Teuchos::Comm<int> >&      comm,
```

Comment:
Argh i forgot one thing: this should be `Teuchos::RCP<const Teuchos::Comm<int>>`.  But if it compiles and passes the test, don't worry about it for now.

Reply:
@mhoemmen just to clarify, the only change here would be to make this a const, right?

That's easy enough to do and probably best to just do today before I forget about it over the break.  I can throw that in and shoot off a new PR for it.

Reply:
@mhoemmen I made this change and put it into PR #4120